tests/tests/scissor_tests/mod.rs

```diff
@@ -0,0 +1,216 @@
+use std::mem::size_of;
+
+use wgpu::util::DeviceExt;
+use wgpu_test::{initialize_test, TestParameters, TestingContext};
+use wgt::BufferAddress;
+
+struct Rect {
+    x: u32,
+    y: u32,
+    width: u32,
+    height: u32,
+}
+
+fn scissor_test_impl(ctx: &TestingContext, scissor_rect: Rect, expected_data: [u8; 16]) {
+    let vertex_data: [f32; 12] = [
+        -1.0_f32, -1.0_f32, 0.0_f32, 1.0_f32, -1.0_f32, 0.0_f32, 1.0_f32, 1.0_f32, 0.0_f32,
+        -1.0_f32, 1.0_f32, 0.0_f32,
+    ];
+    let index_data: [i16; 6] = [0, 1, 3, 1, 3, 2];
```

Comment:
If you're just trying to make fullscreen geometry, you can use the same VS as mipmap does here: https://github.com/gfx-rs/wgpu/blob/trunk/examples/mipmap/src/blit.wgsl

Reply:
Thank you for mentioning that. I wasn't familiar with that technique. Utilizing it allowed me to simplify the test a lot.